contracts/pallet-ibc/routing.md

```diff
@@ -0,0 +1,288 @@
+## Routing (ICS26) and callback handlers
+
```

Comment:
I've changed the router implementation, please look at `master` so you can update the docs

Reply:
Looking at it, we also need a similar implementation for the benchmarks.